lib/slim-rails/register_engine.rb

```diff
@@ -0,0 +1,30 @@
+module Slim
+  module Rails
+    module RegisterEngine
+      def self.register_engine(app, config)
+        if ::Rails::VERSION::MAJOR == 3
+          register_engine3(app, config)
+        else
+          register_engine4(app, config)
```

Comment:
I already use slim-rails with Rails 5. No worries, you can safely use the generic register_engine name for both 4 and 5.